server/src/main/java/com/cloud/hypervisor/KVMGuru.java

```diff
@@ -139,7 +185,7 @@ protected void configureVmOsDescription(VirtualMachine virtualMachine, VirtualMa
             guestOsMapping = _guestOsHypervisorDao.findByOsIdAndHypervisor(guestOS.getId(), getHypervisorType().toString(), hostVo.getHypervisorVersion());
         }
 
-        if (guestOsMapping == null || hostVo == null) {
+        if (guestOsMapping == null) {
```

Comment:
Could this have regression of some kind? cc @fermosan @DaanHoogland

Reply:
HostVo can never be null at this point asfaik.

Reply:
@fermosan 
I read the code from main branch and it seems this can be null if hostId is null in caller at line 126 https://github.com/apache/cloudstack/blob/main/server/src/main/java/com/cloud/hypervisor/KVMGuru.java#L123